### ISSUE TITLE
[DOC-FIX] Document models:/ URIs explicitly in OSS MLflow docs

### ISSUE THREAD

**Issue by smurching** [mlflow/mlflow#6195]
### Willingness to contribute

No. I cannot contribute a documentation fix at this time.

### URL(s) with the issue

https://mlflow.org/docs/latest/tracking.html#artifact-stores

### Description of proposal (what needs changing)

I wanted to link to what an “MLflow model URI” could be in a design doc, and searched for it in MLflow docs. It looks like we don’t actually describe supported model URIs except in [CLI docs](https://mlflow.org/docs/latest/cli.html#cmdoption-mlflow-azureml-build-image-m) which say

```
Required URI to the model. A local path, a 'runs:/' URI, or a remote storage URI (e.g., an 's3://' URI). For more information about supported remote URIs for model artifacts, see https://mlflow.org/docs/latest/tracking.html#artifact-stores
```

However the [linked artifacts doc page ](https://mlflow.org/docs/latest/tracking.html#artifact-stores)doesn’t say anything about models:/ URIs. Currently, it looks like we document models:/ URIs in the model registry page: https://mlflow.org/docs/latest/model-registry.html#fetching-an-mlflow-model-from-the-model-registry.

To address this, one idea is to document model registry `models:/` artifact storage and `runs:/` artifact storage in https://mlflow.org/docs/latest/tracking.html#artifact-stores. We could also add a more explicit section documenting models:/ URIs in the model registry doc page and link to it from the tracking page

**Comment by mrhimanshu** [comment]
Is anyone working on this ? If not then, I can take it up.

**Comment by dbczumar** [comment]
@mrhimanshu Apologies for the delay. We would be very excited if you could contribute a pull request for these documentation improvements. Please let me know if you have any questions.

**Comment by dbczumar** [comment]
Hi @mrhimanshu, are you still interested in working on this item?

**Comment by Nikhil-Mudgal** [comment]
Hi @dbczumar , Can i take up this issue, I am new to open source contributing and I think this might be a good issue to start with.

**Comment by Nikhil-Mudgal** [comment]
Thanks @dbczumar. As per the OP's request where he has proposed 2 solutions to the issue. Carrying out the 2nd one will surely be better as we will be giving providing an explicit section for `models:/` URI in the model registry doc page itself and a link to it from Model Tracking page too. Should i go ahead with that? And can you please provide me link on how to setup the required environment for documentation, it will be great as a headstart?

**Comment by dbczumar** [comment]
@Nikhil-Mudgal I agree, the second solution is better. The following section of the contributing guide should help: https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.rst#writing-docs

**Comment by AbinayaM02** [comment]
Hi @dbczumar: I see that there is no PR on this issue. If it's still open, I would like to pick it up.

**Comment by dbczumar** [comment]
Hi @AbinayaM02 , that sounds great! Thanks so much! I've assigned this issue to you :)

**Comment by AbinayaM02** [comment]
Hi @dbczumar: As per the thread, I'm planning to add the model URI reference to the model registry page, specifically under concepts section https://mlflow.org/docs/latest/model-registry.html#concepts. Since model URIs are used only in the fetching and serving models section in the model registry page, repeating it twice seems unnecessary. Instead, the model URI formats can be added as a concept so that it can be linked at the relevant subsections and other necessary places referred in the original issue. Do you recommend any specific way of doing it?

**Comment by rajveer43** [comment]
Can I work on this?
